src/libraries/System.Private.CoreLib/gen/IntrinsicsInSystemPrivateCoreLibAnalyzer.cs

```diff
@@ -0,0 +1,533 @@
+
+        private static INamespaceSymbol GetNamespace(IAssemblySymbol assembly, params string[] namespaceNames)
+        {
+            INamespaceSymbol outerNamespace = assembly.GlobalNamespace;
+
+            string newFullNamespaceName = "";
+            INamespaceSymbol? foundNamespace = null;
+
+            foreach (var namespaceName in namespaceNames)
+            {
+                if (newFullNamespaceName == "")
+                    newFullNamespaceName = namespaceName;
+                else
+                    newFullNamespaceName = newFullNamespaceName + "." + namespaceName;
+
+                foundNamespace = null;
+
+                foreach (var innerNamespace in outerNamespace.GetNamespaceMembers())
+                {
+                    if (innerNamespace.Name == namespaceName)
+                    {
+                        foundNamespace = innerNamespace;
+                        break;
+                    }
+                }
+
+                if (foundNamespace == null)
+                {
+                    throw new Exception($"Not able to find {newFullNamespaceName} namespace");
+                }
+
+                outerNamespace = foundNamespace;
+            }
+
+            return foundNamespace!;
+        }
+
+        private static IEnumerable<INamedTypeSymbol> GetNestedTypes(INamedTypeSymbol type)
+        {
+            foreach (var typeSymbol in type.GetTypeMembers())
+            {
+                yield return typeSymbol;
+                foreach (var nestedTypeSymbol in GetNestedTypes(typeSymbol))
+                {
+                    yield return nestedTypeSymbol;
+                }
+            }
+        }
+
+        private static IEnumerable<INamedTypeSymbol> GetSubtypes(INamespaceSymbol namespaceSymbol)
+        {
+            foreach (var typeSymbol in namespaceSymbol.GetTypeMembers())
+            {
+                yield return typeSymbol;
+                foreach (var nestedTypeSymbol in GetNestedTypes(typeSymbol))
+                {
+                    yield return nestedTypeSymbol;
+                }
+            }
+
+            foreach (var namespaceMember in namespaceSymbol.GetNamespaceMembers())
+            {
+                foreach (var typeSymbol in GetSubtypes(namespaceMember))
+                {
+                    yield return typeSymbol;
+                }
+            }
+        }
+
+        private sealed class IntrinsicsAnalyzerOnLoadData
+        {
+            public IntrinsicsAnalyzerOnLoadData(List<INamedTypeSymbol> namedTypesToBeProtected,
+                                                INamedTypeSymbol? bypassReadyToRunAttribute,
+                                                INamedTypeSymbol? bypassReadyToRunForIntrinsicsHelperUse)
+            {
+                NamedTypesToBeProtected = namedTypesToBeProtected;
+                BypassReadyToRunAttribute = bypassReadyToRunAttribute;
```

Comment:
It looks like this PR also deletes the `BypassReadyToRunAttribute` type. Is this path that checks for it in the analyzer used?

Reply:
It isn't, but the attribute is still there and effective. In case we need it, I don't want to rev the analyzer.